langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiTokenizer.java

```diff
@@ -114,7 +114,7 @@ private int estimateTokenCountIn(UserMessage userMessage) {
             }
         }
 
-        if (userMessage.name() != null && !modelName.equals(GPT_4_VISION_PREVIEW.toString())) {
+        if (userMessage.name() != null && !modelName.equals(GPT_4_VISION_PREVIEW.toString())) { // TODO: not sure why checking gpt-4-vision-preview, GPT_4_VISION_PREVIEW is deprecated
```

Comment:
deprecated, but still can be used, let's keep it for the moment, it does no harm